testing/testcafe/model/list/index.ts

```diff
@@ -47,6 +47,16 @@ export default class List extends Widget {
     return new ListGroup(this.element.find(`.${CLASS.group}`).nth(index));
   }
 
+  refresh(): Promise<void> {
+    const { getInstance } = this;
+
+    return ClientFunction(
+      // eslint-disable-next-line no-underscore-dangle
+      () => { (getInstance() as any)._refresh(); },
```

Comment:
is it necessary ti use a private api here?

Reply:
https://github.com/DevExpress/DevExtreme/pull/23051/commits/bafbf0d90d2331db45afc885aff652e334a8feb3